testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -101,12 +118,21 @@ public void TestRunTeardown()
         m_TestResultsLocal.Clear();
     }
 
+    private void OnDisable()
```

Comment:
OnDisable or OnDestroy? Cause OnDisable usually means this same instance can be reenabled later.

Reply:
If we do disable, then enable, we'd be in an invalid state no?

Reply:
@NoelStephensUnity can you speak to these changes as I believe you made them?

Reply:
It looks like this is done in OnDestroy as well. I'll ping @NoelStephensUnity about this change.